readme.md

```diff
@@ -1,58 +1,119 @@
-# Pawn
+# Pawn Community Compiler
 
-[![Build Status][build_status]][build] [![Build Status - Windows][build_status_win]][build_win]
+[![Build Status][build_status]][build]
+[![Build Status - Windows][build_status_win]][build_win]
+
+## What
+
+This is a modified copy of the Pawn compiler version 3.2.3664 by Compuphase that
+fixes some bugs and adds a few features.
+
+This project was originally maintained and managed by Zeex who left the project
+due to lack of time on 2017-12-31. Thank you for all your hard work on keeping
+this project alive Zeex!
+
```

Comment:
I am not really sure if @Zeex has fully given up on this project. I think it would be sufficient if it states that Zeex was the founder of this project or something along those lines (just my opinion) with some bits of history.

Reply:
Yes this is rather outdated since it was written when Zeex handed off the repo to me.